apis/config/v1alpha1/config_types.go

```diff
@@ -70,6 +73,10 @@ type ReadinessSpec struct {
 	StatsEnabled bool `json:"statsEnabled,omitempty"`
 }
 
+type Mutation struct {
```

Comment:
Let's not add mutation as a config resource -- flags are easier and will suffice for feature gating dev work